src/lib/database/schema.ts

```diff
@@ -29,6 +31,11 @@ export interface DatabaseMethods {
     created_at?: string
     workspace_id?: string
   }) => Workspace
+  addAPIKey: (params: {
```

Comment:
It has to be like this since this is our name for `api_key` not our `api_k_e_y`: otherwise we break snake case to param case transformations (if you transform then transform back you get a different result).

```suggestion
  addApiKey: (params: {
```

Reply:
I'm ok with establishing that as the Seam convention, but we have historically used airbnb convention https://twitter.com/seveibar/status/1585389759303938048?s=20

Reply:
Airbnb convention in case you aren't on twitter haha

![image](https://github.com/seamapi/fake-seam-connect/assets/1910070/f4ab5ce3-04b9-4f6a-86ca-8ed0e55694cf)

Reply:
Also note that microsoft has a different convention for two letter acronyms. The convention you're suggesting is the Google convention I think, which I'm fine with since our internal usage of snake case makes anything else pretty annoying, and I'm a bit indifferent

Reply:
Also I guess I should also say I should probably respect the convention of the project lol- so not super relevant, but always fun to talk about camel case

Reply:
Yes, but IMO there are two competing conventions. With `HTTPRequest` there is no `http_request` or `seam.httpRequest` method. It's just the name of the idea. 

Since Seam is highly concerned with supporting the SDK in various languages, I think a stronger convention is the unambiguous transformation between underscore and camel case. At least from my POV, it is frustrating having to guess what _might_ be an acronym or not (actually Initialism).

This thread may not be the best forum for this particular discussion though. Fun fact, golang basically forces you to use this convention for `ID` https://github.com/golang/go/wiki/CodeReviewComments#initialisms. If we followed this convention consistently, then we would always write `fooID`. If you only live in Go, then you don't really notice this too much (unless on the edges of api boundaries that use `fooId`).

Reply:
6f1dbf5